README.md

```diff
@@ -123,6 +123,16 @@ cp /usr/i686-w64-mingw32/bin/lib{gcc_s_dw2-1,ssp-0,stdc++-6,winpthread-1}.dll /m
 wine /my/path/to/Winamp/winamp.exe
 ```
 
+## Building & Running on Windows
+
+Open the folder with Visual Studio, it should automatically detect the CMake configurations.
+
+If you don't want to do this, there are some caveats to using CMake itself directly:
+
+* Use -DCMAKE_GENERATOR_PLATFORM=Win32 to get a 32-bit project (Nothing builds under 64-bit with the MS compiler due to __asm blocks (yet))
+* CMake GUI doesn't work and doesn't tell you why it doesn't work. It is recommended to avoid it.
+* CMake doesn't rebuild the project very well if you don't clean out the generated files first. Visual Studio handles this for you.
+* There are crashes in evallib using the latest compilers, which rules out pretty much any AVS preset from working (currently)
```

Comment:
I still think this should move to a note at the top of the section, like in my previous comment on this line.